django/db/migrations/autodetector.py

```diff
@@ -521,6 +529,9 @@ def generate_created_models(self):
                     related_fields[field.name] = field
                 if getattr(field.remote_field, "through", None) and not field.remote_field.through._meta.auto_created:
                     related_fields[field.name] = field
+            # Are there any indexes to defer?
```

Comment:
This comment doesn't really make sense. Could you explain why you're doing what you're doing?